open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ax = df[df.index&lt;"2020-05-01"]['Rt'].______(np.inf,np.nan).______(method='pad').plot(figsize=(10,3))
</pre><p>Please follow this to format all the python code, https://stackoverflow.com/questions/65747615/how-to-format-jupyter-notebook-in-vscode</p><p> </p><p>def get_rt_ax() for the code before .plot</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Finished it</p>